tests/common/include/tests/common/material_interaction.inl

```diff
@@ -384,18 +384,22 @@ TEST(material_interaction, telescope_geometry_energy_loss) {
 TEST(material_interaction, telescope_geometry_scattering_angle) {
     vecmem::host_memory_resource host_mr;
 
-    // Use rectangle surfaces in the telescope
-    mask<rectangle2D<>> rectangle{0u, 2000.f * unit<scalar>::mm,
-                                  2000.f * unit<scalar>::mm};
-
     // Build in x-direction from given module positions
     detail::ray<transform3> traj{{0.f, 0.f, 0.f}, 0.f, {1.f, 0.f, 0.f}, -1.f};
-    std::vector<scalar> positions = {0.f, 1000.f * unit<scalar>::cm,
-                                     2000.f * unit<scalar>::cm};
+    std::vector<scalar> positions = {0.f};
 
+    // To make sure that someone won't put more planes than one by accident
+    EXPECT_EQ(positions.size(), 1u);
```

Comment:
In this case, it should be possible to call the telescope creation like this:
```
const auto det = create_telescope_detector(host_mr, rectangle, 1u);
```

Reply:
```tel_length``` is should not be used here anyway...

Reply:
but the argument itself is positioned between rectangle and material properties - so I need to put it anyway?

Reply:
Ah, yes, sorry, in that case you are right. Let me put this in a config struct at some point...